dataactvalidator/config/sqlrules/a18_cross_file.sql

```diff
@@ -1,10 +1,26 @@
 SELECT
-    approp.row_number,
-    approp.tas,
-    approp.gross_outlay_amount_by_tas_cpe,
-    SUM(op.gross_outlay_amount_by_pro_cpe) as gross_outlay_amount_by_pro_cpe_sum
-FROM appropriation approp JOIN object_class_program_activity op
-ON approp.tas = op.tas AND approp.submission_id = op.submission_id
+	approp.row_number,
```

Comment:
To answer @cmc333333's question about the addition of all the news in the `SELECT` and `GROUP BY` clauses....

This is essentially a bug fix. The new columns added to these clauses are the 7 fields that make up the TAS, the key that joins financial data files (e.g. appropriations, object_class_program_activity, and award_financial) together. 

For internal use, we created a column in each submission table (`tas`) that represents the combined 7 fields, We can use that column to make joins between the tables easier, but we can't use it for external display, which is why this query needed an update.